common/main/java/com/couchbase/lite/internal/core/C4Database.java

```diff
@@ -175,11 +175,13 @@ public static C4Database getDatabase(
         @NonNull String parentDirPath,
         @NonNull String name,
         boolean isFullSync,
+        boolean isMMapEnabled,
         int algorithm,
         @Nullable byte[] encryptionKey)
         throws LiteCoreException {
         int dbFlags = DB_FLAGS;
         if (isFullSync) { dbFlags |= C4Constants.DatabaseFlags.DISC_FULL_SYNC; }
+        if (isMMapEnabled) { dbFlags |= C4Constants.DatabaseFlags.MMAP; }
```

Comment:
This should be !isMMapEnabled as in LiteCore the flag is MMapDisabled.

Reply:
Wow!  Thanks!  Great catch!